app/src/main/java/com/amaze/filemanager/fragments/ProcessViewerFragment.java

```diff
@@ -209,7 +221,7 @@ public void processResults(final DatapointParcelable dataPackage, int serviceTyp
 
             Spanned timerSpan = Html.fromHtml(getResources().getString(R.string.service_timer)
                     + ": <font color='" + accentColor + "'><i>"
-                    + formatTimer(++time)
+                    + Utils.formatTimer(++time)
```

Comment:
I don't understand, why are you doing `++time`, then formating, are you supposing that this function will be called exactly every second? Why aren't you setting a start time and then recalculating the elapsed time?

Reply:
Yes, this function will be called every second. Setting a start time and calculating elapsed time would be waste of resources.

Reply:
PRing explanation.